lib/apia/response.rb

```diff
@@ -20,6 +25,11 @@ def initialize(request, endpoint)
       @headers = {}
     end
 
+    def plain!
+      @type = PLAIN
+      @body = ''
+    end
```

Comment:
I assume the usage pattern within an endpoint is be something like this?

```ruby
response.plain!
response.body = "my awesome response"
```

Do we care about requiring `#plain!` to be called before the body value is set? Might a nicer usage pattern be a helper method? Though I'm not sure what such a method might be called, maybe `respond_with(body, content_type = nil)`? What do you think?